plugins/styleguide/assets/javascripts/discourse/lib/styleguide.js

```diff
@@ -1,11 +1,54 @@
-import DiscourseTemplateMap from "discourse-common/lib/discourse-template-map";
-
 let _allCategories = null;
 let _sectionsById = {};
 let _notes = {};
 
 export const CATEGORIES = ["atoms", "molecules", "organisms"];
 
+const SECTIONS = [
+  "sections/atoms/00-typography",
+  "sections/atoms/01-font-scale",
+  "sections/atoms/02-buttons",
+  "sections/atoms/03-colors",
```

Comment:
To make this embroider/tree-shaking compatible, we'll need to pass components by reference. So in this case it would be something like

```js
import TypographyComponent from "../components/sections/atoms/00-typography";
import FontScaleComponent from "../components/sections/atoms/01-font-scale";
import ButtonComponent from "../components/sections/atoms/02-buttons";
...

const SECTIONS = [
  TypographyComponent,
  FontScaleComponent,
  ButtonsComponent,
  ...
];
```

And then rendered using anglebracket syntax like

```hbs
{{#each this.sections as |section|}}
  <section />
{{/each}}
```
 (and the plugin API should also accept component references)